### PR TITLE
develop: add develop docs OWNERS

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by sykp241095 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Comment by wuhuizuo** [comment]
We can create a `OWNERS_ALIASES` file to setup the group, and use alias to assign reviewers and approvers.
@Mini256

File: develop/OWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+reviewers:
+  - Icemap
+  - sykp241095
+  - shizn
+  - winkyao
+  - shczhen
+  - hooopo
+  - Mini256
+  - wd0517
+  - it2911
```
[the remark this answers is not in the document]
Comment (wuhuizuo) [reply develop/OWNERS:16]: `/OWNERS_ALIASES`  example: https://github.com/pingcap/tidb/blob/master/OWNERS_ALIASES

File: develop/OWNERS [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+  - Oreoxmt
+  - qiancai
+  - ran-huang
```
Comment (wuhuizuo) [inline develop/OWNERS:5]: Also we can set a alias group for the members.

**Comment by sykp241095** [comment]
@qiancai Can you help merge this pr :-)

**Comment by ran-huang** [comment]
approve

**Pull request merged**